Sklep/Utils/BarcodeScanner.cs

```diff
@@ -45,8 +46,9 @@ private void VideoCaptureDevice_NewFrame(object sender, AForge.Video.NewFrameEve
             {
                 pictureBox.Image = bitmap;
             }
-            if(result != null && EANValidator.validateBarcode(result.ToString()))
+            if(result != null && EANValidator.validateBarcode(result.ToString()) && DateTime.Now.Subtract(lastScanTime).TotalSeconds > 2)
```

Comment:
No ok, tylko, że teraz blokujesz skanowanie na 2 sekundy czegokolwiek. Zamysł był taki, żeby 2 sekundowy cooldown liczył się tylko przy skanowaniu tego samego produktu. Należy dodać coś w stylu `lastScannedBarcode` i jeśli obecny barcode jest taki sam, to upewnić się, że minęły 2 sekundy. Chociaż w przypadku naszej kamerki 2 sekundy to i tak na tyle mało, że raczej tak szybko skanować nie będziemy, to warto chyba nie blokować potencjalnie możliwości szybszego skanowania.